### DISCUSSION TITLE
Well Done! You're making good progress!

### DISCUSSION THREAD

**Discussion by antonycartwright** [dotnet/maui#23600] [General]
Well done everyone involved. I came about 18 months ago, and Maui was barely usable. I came back 6 months ago, it was usable but with annoying glitches, controls not working as expected, etc. I came back yesterday with a new project, (ok there was some pain for an hour at the start) but it's now pretty impressive.

Good job all! :-)

**Comment by MhAllan** [comment]
impressive until you do something serious. till now you can't use native libs, re-invent the wheels if you can.
